paparazzi/build.gradle

```diff
@@ -72,8 +72,12 @@ subprojects {
       }
     }
 
-    mavenPublish {
-      releaseSigningEnabled = hasProperty('signingKey')
+    plugins.withId('signing') {
+      signing {
+        def signingKey = findProperty('signingKey')
+        def signingPassword = ''
+        useInMemoryPgpKeys(signingKey, signingPassword)
```

Comment:
The plugin has something built in for picking up in memory signing keys you just need to rename your `signingKey` property to `signingInMemoryKey` https://github.com/vanniktech/gradle-maven-publish-plugin#signing